shared/reducers/wallets.js

```diff
@@ -263,6 +263,8 @@ export default function(state: Types.State = initialState, action: WalletsGen.Ac
       return state.merge({
         acceptingDisclaimerDelay: true,
       })
+    case WalletsGen.loadedMobileOnlyMode:
+      return state.update('mobileOnlyMap', map => map.set(action.payload.accountID, action.payload.enabled))
```

Comment:
should be
```
return state.setIn(['mobileOnlyMap', action.payload.accountID], action.payload.enabled)
```

Reply:
ah ok, i tried `return state.updateIn(['mobileOnlyMap', action.payload.accountID], action.payload.enabled)` before which didn't work and i didn't know why. `setIn` is fine, thanks!